server/src/test/java/org/elasticsearch/index/engine/ShuffleForcedMergePolicyTests.java

```diff
@@ -34,6 +34,7 @@ public class ShuffleForcedMergePolicyTests extends BaseMergePolicyTestCase {
     public void testDiagnostics() throws IOException {
         try (Directory dir = newDirectory()) {
             IndexWriterConfig iwc = newIndexWriterConfig();
+            iwc.setMaxFullFlushMergeWaitMillis(0L);
```

Comment:
nit: can you add a comment mentioning that `0L` disables merging on full flush?

Reply:
Done!